tests/platform_tests/api/test_psu.py

```diff
@@ -0,0 +1,109 @@
+import logging
+import re
+
+import pytest
+import yaml
+
+from common.helpers.assertions import pytest_assert
+from common.helpers.platform_api import psu
+from common.helpers.platform_api import chassis
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.sanity_check(skip_sanity=True),
+    pytest.mark.disable_loganalyzer  # disable automatic loganalyzer
```

Comment:
Recently there was a new 'topology' marker added. Please mark these tests to run on 'any' topology. See https://github.com/Azure/sonic-mgmt/blob/master/tests/platform_tests/api/test_chassis.py#L14 for an example.